src/api/users/src/routes/users.js

```diff
@@ -28,27 +28,41 @@ router.get('/:id', validateId(), async (req, res, next) => {
 });
 
 // get all users
-// rejected if the user collection is empty, otherwise users returned
-router.get('/', async (req, res, next) => {
-  try {
-    const usersRef = db.collection('users');
-    const snapshot = await usersRef.get();
+// rejected if the users collection is empty
+// otherwise n (specified via params) users are returned
+router.get('/', validateGetParams(), async (req, res, next) => {
+  // if req.query.page is 0 or unparsable set to 1, otherwise set to the req.query.page
+  const parseReqPage = parseInt(req.query.page, 10);
+  const page = !parseReqPage ? 1 : parseReqPage;
```

Comment:
What happens here if I enter a -1 for page? Should we add a  condition that checks for negative values or 0?

Reply:
Great question! Entering a negative number here returns the first page, and I think this behaviour is okay (I'm actually not sure why this works)

Reply:
Should figure out if this behavior is cause of the db or something else. I feel like code shouldn't just "magically work" because of a idiot-proof mechanic that might only be specific to that tool. If this behavior is caused the result of a specific db, what happens if in the future we switch dbs to say mongo and this breaks?

Reply:
This should be taken care of since I just hardcoded the values to not accept numbers < 1